### ISSUE TITLE
Move multiple issues into a milestone

### ISSUE THREAD

**Issue by azenMatt** [ZenHubIO/support#561]
Similar to #118 but for milestones. 

cc @Nowaker

**Comment by Anvil102** [comment]
Would definitely like to see this one make it for Q3 2016. With how we plan sprints at Leap, we often end up with a huge backlog of issues that don't make it into the final commitment for a sprint .We then have to move one backlog issue at a time to the next sprint (i.e. milestone). Being able to select multiple issues in a pipeline and batch move them to a milestone would be a god send.

**Comment by wdelstrother** [comment]
The absence of this feature is one of the main reasons we've decided not to use zenhub; having to modify issues one at a time in the board view is a massive pain, and makes sprint planning very slow.  Adding this feature would be a massive boon for us.  :+1

**Comment by azenMatt** [comment]
This change is now live 💥 

https://www.zenhub.com/blog/task-board-bulk-actions/

**Comment by Anvil102** [comment]
Great to see @azenMatt!! I'll be trying this out soon!

**Comment by 8purkil** [comment]
Hi Is the feature enabled in the Enterprise version?   I am using Zen Hub enterprise on Mac and cannot see either of the options to select multiple issues.  Thanks

**Comment by azenMatt** [comment]
Hi @8purkil - not quite yet. The Enterprise team are just completing the final testing of this release to be sent out at the start of next week. So it should be with you very soon!

**Comment by ride4sun** [comment]
is it done? If yes - where is it?

**Comment by azenMatt** [comment]
Hi @ride4sun the details are here - https://www.zenhub.com/blog/task-board-bulk-actions/

To access this, in the board view you can either: 

1) select the avatar of a card 
2) use the 'cog' menu to select multiple as an option

**Comment by valentinweber** [comment]
Hi @azenMatt.

I recently discovered this feature which saves me a lot of manual work.

I've noticed that filters can be used to select a set of issues in one click.
![image](https://user-images.githubusercontent.com/9213710/50762511-3f35f680-12c1-11e9-9bbc-122076d8d679.png)

However I couldn't find a easy way to select all the issues in a given pipeline (ideally after applying a filter) and currently I'm still selecting them one by one. Am I missing something?
